hack/convert-output-to-tsv.py

```diff
@@ -1,175 +1,157 @@
+    elif table_name == "hles_cancer_condition":
+        pk_name = 'dog_id'
+        entity_name = f"{PRIMARY_KEY_PREFIX}:{table_name}_id"
+    # hles_health_condition: read + copy dog_id and hs_condition and hs_condition_is_congenital
+    # concatenate and write out as pk_name
+    elif table_name == "hles_health_condition":
+        pk_name = None
+        entity_name = f"{PRIMARY_KEY_PREFIX}:{table_name}_id"
+    elif table_name == "environment":
+        pk_name = 'dog_id'
+        entity_name = f"{PRIMARY_KEY_PREFIX}:{table_name}_id"
+    else:
+        log.info(f"Unrecognized table: {table_name}")
+        break
+
+    # read json data
+    for path in gcs.ls(os.path.join(parser.input_dir, table_name)):
+        log.info(f"...Opening {path}")
+
+        with gcs.open(path, 'r') as json_file:
+            for line in json_file:
+                row = json.loads(line)
                 # hles_health_condition: read + copy dog_id and hs_condition and hs_condition_is_congenital
                 # concatenate and write out as pk_name
-                elif table_name == "hles_health_condition":
-                    entity_name = pk_prefix + table_name + '_id'
+                if table_name == "hles_health_condition":
                     # grab the congenital flag and convert to int
                     congenital_flag = row.get('hs_condition_is_congenital')
-                    if congenital_flag == True:
-                        congenital_flag = 1
-                    elif congenital_flag == False:
-                        congenital_flag = 0
-                    else:
-                        print (f"Error, 'hs_condition_is_congenital' is not populated in {table_name}")
-                    row[entity_name] = ('%s-%s-%s' % 
-                        (row.get('dog_id'), row.get('hs_condition'), congenital_flag))
-                    # store data
-                    column_set.update(row.keys())
-                    row_list.append(row)
-                # environment:
+                    try:
+                        congenital_flag = int(congenital_flag)
+                    except TypeError:
+                        log.info(f"Error, 'hs_condition_is_congenital' is not populated in {table_name}")
+                    row[entity_name] = '-'.join([row.get('dog_id'), row.get('hs_condition'), congenital_flag])
                 elif table_name == "environment":
-                    entity_name = pk_prefix + table_name + '_id'
-                    # env records are unique on dog_id, redcap_event
-                    # setting the primary key
                     dogId = row.get('dog_id')
                     eventName = row.get('address_month_year')
-                    row[entity_name] = ('%s-%s' %
-                        (dogId, eventName))
-                    column_set.update(row.keys())
-                    row_list.append(row)
+                    row[entity_name] = '-'.join([dogId, eventName])
```

Comment:
I get a the following TypeError when testing this out:
`*** TypeError: sequence item 0: expected str instance, int found`

I believe dogId is an int being parsed as an int and will need to be cast to string.